reader.go

```diff
@@ -115,27 +116,28 @@ func (r *Reader) csvAddressFile() error {
 	defer f.Close()
 
 	// Read File into a Variable
-	lines, err := csv.NewReader(f).ReadAll()
-	if err != nil {
-		return err
-	}
-
-	// Loop through lines & turn into object
-	for _, csvLine := range lines {
-		//csvLine := replaceNull(csvLine)
-
-		a := Address{
-			EntityID:                    csvLine[0],
-			AddressID:                   csvLine[1],
-			Address:                     csvLine[2],
-			CityStateProvincePostalCode: csvLine[3],
-			Country:                     csvLine[4],
-			AddressRemarks:              csvLine[5],
+	reader := csv.NewReader(f)
```

Comment:
cc @bkmoovio I had to do this so we could ignore the last line, which contained a whitespace character. For some reason `ReadAll()` would error and fail the whole `Read()`.

Reply:
Yeah as we spoke about:... the white space character isn't defined in the sdn specification,  and I agree with you that I don't think that will be fixed by the treasury department anytime soon.